db/migrate/20241011194431_create_preferences.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePreferences < ActiveRecord::Migration[7.1]
+  def change
+    create_table :preferences do |t|
+      t.string :name
```

Comment:
It is a good practice to keep your model and database logic in sync, ie, have validations in the database as the model has. Example, no null here for the name attribute.